### PR TITLE
CLOUD-481 Chart upgrade due to new version release

### PR DESCRIPTION
**Pull request by cap1984**

### CODE REVIEW

File: charts/pxc-db/Chart.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
-apiVersion: v1
-appVersion: "1.1.0"
+apiVersion: v1-3-0
```
Comment (paulczar) [inline charts/pxc-db/Chart.yaml]: this change of API version is probably going to cause issues, this is the version for the Helm API.  see https://helm.sh/docs/topics/charts/#the-apiversion-field

File: charts/pxc-db/templates/cluster.yaml [hunk 2]
```diff
@@ -1,6 +1,9 @@
-apiVersion: pxc.percona.com/v1
+apiVersion: pxc.percona.com/{{ .Chart.ApiVersion }}
```
Comment (paulczar) [inline charts/pxc-db/templates/cluster.yaml]: see previous comment.

File: charts/pxc-db/templates/cluster.yaml [hunk 3]
```diff
@@ -1,6 +1,9 @@
...
 kind: PerconaXtraDBCluster
 metadata:
+  annotations:
+    kubectl.kubernetes.io/last-applied-configuration: |
+        {"apiVersion":"pxc.percona.com/{{ .Chart.ApiVersion }}","kind":"PerconaXtraDBCluster"}
```
Comment (paulczar) [inline charts/pxc-db/templates/cluster.yaml]: normally you wouldn't see a last-applied-configuration and let Kubernetes do that itself.   What's the reason for adding this here ?

**Comment by paulczar** [comment]
I'm not 100% sure the comments in my review are the case of the CI failures.

Helm has also changed its recommendation about CRDs, saying to pull them out of templates and put them untemplated in a seperate `crd` directory, I would recommend updating this to do that.

https://helm.sh/docs/topics/chart_best_practices/custom_resource_definitions/#method-1-let-helm-do-it-for-you

File: charts/pxc-db/templates/cluster.yaml (hunk 3 above)
Comment (cap1984) [reply charts/pxc-db/templates/cluster.yaml]: We are facing the following error without the annotations 

`    - lastTransitionTime: "2020-01-20T18:48:17Z"
      message: 'wrong PXC options: set version: new version: Malformed version: '
      reason: ErrorReconcile
      status: "True"
      type: Error`

File: charts/pxc-db/Chart.yaml (hunk 1 above)
Comment (cap1984) [reply charts/pxc-db/Chart.yaml]: done

File: charts/pxc-db/templates/cluster.yaml (hunk 2 above)
Comment (cap1984) [reply charts/pxc-db/templates/cluster.yaml]: done

**Pull request merged**